configs/_base_/models/resnet50_cifar_cutmix.py

```diff
@@ -0,0 +1,16 @@
+# model settings
+model = dict(
+    type='ImageClassifier',
+    backbone=dict(
+        type='ResNet_CIFAR',
+        depth=50,
+        num_stages=4,
+        out_indices=(3, ),
+        style='pytorch'),
+    neck=dict(type='GlobalAveragePooling'),
+    head=dict(
+        type='MultiLabelLinearClsHead',
+        num_classes=10,
+        in_channels=2048,
+        loss=dict(type='CrossEntropyLoss', loss_weight=1.0, use_soft=True)),
+    train_cfg=dict(cutmix=dict(alpha=1.0, num_classes=10)))
```

Comment:
@HIT-cwh The `cutmix_prob` seems to be missing here.

Reply:
Great point. Will fix ASAP